src/FuncSharp/Option/ValueOptionExtensions.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace FuncSharp
+{
+    public static class ValueOptionExtensions
+    {
+        /// <summary>
+        /// Returns value of the option if it has value. If not, returns null.
+        /// </summary>
+        public static A GetOrNull<A>(this ValueOption<A> option)
+            where A : class
+        {
+            return option.GetOrDefault();
+        }
+
+        /// <summary>
+        /// Returns value of the option if it has value. If not, returns zero.
+        /// </summary>
+        public static int GetOrZero(this ValueOption<int> option)
+        {
+            return option.GetOrDefault();
+        }
+
+        /// <summary>
+        /// Returns value of the option if it has value. If not, returns zero.
+        /// </summary>
+        public static decimal GetOrZero(this ValueOption<decimal> option)
+        {
+            return option.GetOrDefault();
+        }
+
+        /// <summary>
+        /// Returns value of the option if it has value. If not, returns false.
+        /// </summary>
+        public static bool GetOrFalse(this ValueOption<bool> option)
+        {
+            return option.GetOrDefault();
+        }
+
+        /// <summary>
+        /// Returns value of the option if it has value. If not, returns the <paramref name="otherwise"/>.
+        /// </summary>
+        public static B GetOrElse<A, B>(this ValueOption<A> option, B otherwise)
+            where A : B
+        {
+            if (option.NonEmpty)
+            {
+                return option.GetOrDefault();
+            }
+            return otherwise;
+        }
+
+        /// <summary>
+        /// Returns value of the option if it has value. If not, returns value created by the otherwise function.
+        /// </summary>
+        public static B GetOrElse<A, B>(this ValueOption<A> option, Func<Unit, B> otherwise)
+            where A : B
+        {
+            if (option.NonEmpty)
+            {
+                return option.GetOrDefault();
+            }
+            return otherwise(Unit.Value);
+        }
+
+        /// <summary>
+        /// Returns the option if it has value. Otherwise returns the alternative option.
+        /// </summary>
+        public static ValueOption<B> OrElse<A, B>(this ValueOption<A> option, Func<Unit, ValueOption<B>> alternative)
+            where A : B
+        {
+            return option.Match(
+                v => ValueOption.Valued((B)v),
+                _ => alternative(_)
```

Comment:
I think we usually dont pass the unit. 🤔 
```suggestion
                _ => alternative(Unit.Value)
```

Reply:
Yeah, but I figured there's no reason to ask for another one, when we already have one. 😄